src/components/Loader.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import "../style.css";
```

Comment:
You don't need to import this here, it's already imported once in the `src/index.js`